src/main/java/org/hyperskill/hstest/stage/SpringTest.java

```diff
@@ -172,12 +181,34 @@ public static void startSpring() throws Exception {
                                 .map(Class::getCanonicalName)
                                 .collect(Collectors.joining(", "))
                 );
+            } else if (!ReflectionUtils.hasMainMethod(suitableClasses.get(0)) && !isKotlin) {
+                throw new ErrorWithFeedback("The main method was not found in the class");
```

Comment:
The better feedback would be 
`"The main method was not found in the class " + suitableClasses.get(0).getSimpleName() `.

This way, users will understand which class should have main method. Otherwise it's not clear.